_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -280,6 +297,7 @@ class FallbackProxyENCODE : public VideoENCODE
                         bool isPlatformHardware)
         : m_core(core), m_coreInt(coreInt), m_codecHandlers(handlers)
         , m_isPlatformHardware(isPlatformHardware)
+        , m_useSWFallback(false)
```

Comment:
Can we use [core->GetPlatfromType] instead of [m_isPlatformHardware] flag?

Reply:
Thanks!